app/models/versioned/edition.rb

```diff
@@ -84,6 +84,44 @@ def self.create_initial(document, user = nil, tags = {})
               status: status)
     end
 
+    def self.create_next_edition(proceeding_edition, user)
+      revision = proceeding_edition.revision.build_next_revision(
+        { change_note: "", update_type: "major" },
+        user,
+      )
+
+      status = EditionStatus.create!(created_by: user,
+                                     revision_at_creation: revision,
```

Comment:
This is used so that when someone wants to compare a document based on a state change it knows what state the revision was at the time of the state change.

Reply:
👍